.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: node_js
 node_js:
   - "node"
+  - "10"
```

Comment:
You said `node` is set to `10`, do we need to add `9`

Reply:
`node` is set to the latest version. So for now it's `10` but when `11` comes out, it'll be that. The specific ones I call out are those that have long-term support (LTS) which are the evens

Reply:
why add 10 if node is already at 10?

Reply:
Because once newer versions come out, `node` will no longer be 10. It's how I caught the issue with Node 10 so quickly. `node` became 10 when it was released and broke the build. My thinking is that it'd be good to make sure that the SDK can always run on the latest version of Node

Reply:
So `node` will auto catch 11 when it comes out? Or 12?

Reply:
Whatever the latest version is. So 11, then 12, etc...